style.css

```diff
@@ -0,0 +1,180 @@
+/* stylelint-disable function-calc-no-unspaced-operator */
+/* stylelint-disable no-missing-end-of-source-newline */
+/* stylelint-disable color-hex-length */
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+/* stylelint-disable color-hex-case */
+/* stylelint-disable length-zero-no-unit */
+/* stylelint-disable indentation */
+
```

Comment:
- [ ] You currently have unused commented code, this doesn't make your project professinal kindly remove them
- [ ] I also understand that removing them has the tendency to make your linters not pass, however to fix this issue, kindly run these commands in your vscode terminal and they should automatically six all of your stylelint linters `npx stylelint "**/*.{css,scss}" --fix` :+1: